### PR TITLE
5: Content type projects.

### PR DESCRIPTION
**Pull request by robherba**
**Steps to test**
- Run the script manatime_local_install.sh.
- See if the features manatime_feature_projects was added and activated.
- Add a new project node.

**DoD**
- As an administrator I should be able to go to node/add/project and add a new project with the described fields
- As an administrator, I should be able to view a project (no themed) and click on the Edit tab and update that project.

[issue link](https://github.com/ManatiCR/manatime/issues/5)

### CODE REVIEW

**Comment by robherba** [comment]
I'm going to rename the field name from body to description.

File: drupal.make.yml [hunk 1]
```diff
@@ -47,6 +47,8 @@ projects:
     version: ""
   fences:
     version: ""
+  inline_entity_form:
+    version: ""
```
Comment (kporras07) [inline drupal.make.yml]: Please write a version number

**Comment by kporras07** [comment]
@rohernandez  2 minor changes:
1) Use multiple values widget always (although the field only supports 1 value): 
screenshot from 2016-08-24 16 01 58

2) Override labels: 
screenshot from 2016-08-24 16 02 11

**Comment by robherba** [comment]
Changes has been applied.

File: modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc [hunk 2]
```diff
@@ -137,16 +137,16 @@ function manatime_feature_projects_field_default_field_instances() {
         'fields' => array(),
         'type_settings' => array(
           'allow_clone' => 0,
-          'allow_existing' => FALSE,
+          'allow_existing' => 0,
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc]: We need to allow existing.

File: modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc [hunk 3]
```diff
@@ -137,16 +137,16 @@ function manatime_feature_projects_field_default_field_instances() {
...
           'allow_new' => 1,
           'delete_references' => 0,
           'label_plural' => 'nodes',
           'label_singular' => 'node',
           'match_operator' => 'CONTAINS',
-          'override_labels' => 0,
+          'override_labels' => 1,
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc:146]: We need to override the labels. Add new labels

File: modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+      'user_register_form' => FALSE,
+    ),
+    'widget' => array(
+      'active' => 1,
+      'module' => 'inline_entity_form',
+      'settings' => array(
+        'fields' => array(),
+        'type_settings' => array(
+          'allow_clone' => 0,
+          'allow_existing' => 0,
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc]: We need to allow existing. Check it in the config UI.

File: modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+          'allow_new' => 1,
+          'delete_references' => 0,
+          'label_plural' => 'nodes',
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc]: We need to change the labels to client/clients. Check it in the config UI.

**Comment by robherba** [comment]
Changes has been applied.

File: modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc [hunk 6]
```diff
@@ -125,7 +125,7 @@ function manatime_feature_projects_field_default_field_instances() {
     'entity_type' => 'node',
     'fences_wrapper' => 'div',
     'field_name' => 'field_client',
-    'label' => 'Client',
+    'label' => 'Clients',
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.features.field_instance.inc:128]: label should be singular (Client)

**Comment by robherba** [comment]
Changes has been applied.

**Comment by kporras07** [comment]
@rohernandez :shipit:

**Pull request merged**